docs/index.rst

```diff
@@ -201,6 +247,10 @@ contacting support.
 
 .. [5] ``grpc.aio`` is automatically instrumented starting with ``grpcio>=1.32.0``.
```

Comment:
Nit: the numbering is off because it skipped 4

Reply:
fixed.